Wikipedia/assets/index.js

```diff
@@ -1181,38 +1202,40 @@ var newCaption = function newCaption(title, headerText) {
 };
 
 /**
- * @param {!Document} document
+ * @param {!Window} window
  * @param {!Element} content
  * @param {?string} pageTitle
  * @param {?boolean} isMainPage
  * @param {?string} infoboxTitle
  * @param {?string} otherTitle
  * @param {?string} footerTitle
+ * @param {?FooterDivClickCallback} footerDivClickCallback
  * @return {void}
  */
-var collapseTables = function collapseTables(document, content, pageTitle, isMainPage, infoboxTitle, otherTitle, footerTitle, footerDivClickCallback) {
+var collapseTables = function collapseTables(window, content, pageTitle, isMainPage, infoboxTitle, otherTitle, footerTitle, footerDivClickCallback) {
   if (isMainPage) {
     return;
   }
 
   var tables = content.querySelectorAll('table');
-  for (var i = 0; i < tables.length; ++i) {
+
+  var _loop = function _loop(i) {
     var table = tables[i];
 
     if (elementUtilities.findClosestAncestor(table, '.app_table_container') || !shouldTableBeCollapsed(table)) {
-      continue;
+      return 'continue';
```

Comment:
@montehurd, I'm not sure why this changed but it looks correct albeit obfuscated. Looking a couple lines up, this changed from a for loop to a function called loop so the `continue` keyword no longer applies and you can see how it's used further down like you mentioned. If you're feeling doubtful, you might try removing your node_modules directory and reinstalling to verify you have the correct versions of the tools and then just toggling a section.